src/flowcean/core/environment/offline.py

```diff
@@ -35,3 +35,28 @@ def as_stream(self, batch_size: int = 1) -> StreamingOfflineData:
             A streaming offline data.
         """
         return StreamingOfflineData(self, batch_size)
+
+    def stack(self, other: OfflineEnvironment) -> OfflineEnvironment:
+        """Combine this environment with another one vertically.
+
+        Args:
+            other: The environment to append vertically.
+
+        Returns:
+            The combined environment.
+        """
+        # Necessary to prevent circular imports
+        from .stack import StackEnvironment
+
+        return StackEnvironment(self, other)
+
+    def __and__(self, other: OfflineEnvironment) -> OfflineEnvironment:
+        """Combine this environment with another one vertically.
+
+        Args:
+            other: The environment to append vertically.
+
+        Returns:
+            The combined environment.
+        """
+        return self.stack(other)
```

Comment:
I'm not sure about that. This is something like `a_env + b_env`, correct? This syntax is not showing whether it concatenates them vertically or horizontally. This syntax could also combine features.

Reply:
The syntax is `env_a & env_b`. The general idea was to use `|` for horizontal concatenation of two frames. However, I understand that this is not clear. I would suggest to remove the feature for now and have a discussion about it in one of the next meetings?

Reply:
Agreed, sounds good

Reply:
I created the issue #89 to keep track of further discussions.